### PR TITLE
Update outdated `vue` document

### PR DESCRIPTION
**Pull request by KsRyY**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by KsRyY** [comment]
Sorry for that much tries from success :face_with_head_bandage:

**Comment by KsRyY** [comment]
Explaination about what I do: I moved the old `vue` document to `vue-init` and modified some description. I also added the doc of the newly introduced `vue serve` and `vue build` to a new page because these two commands require additional tools.

File: pages/common/vue-init.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# vue-init
```
Comment (sbrl) [inline pages/common/vue-init.md]: Could we change the title so that it matches the format of other subcommand pages like `git merge`, please?
```suggestion
# vue init
```

File: pages/common/vue-init.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+> Legacy init command.
+> Require `vue-cli@2` or `@vue/cli` with `@vue/cli-init`.
```
Comment (sbrl) [inline pages/common/vue-init.md]: I'm not sure we need this line. The point of tldr pages is to show how to _use_ a command, not to install it. I think we can assume that if the user is looking up the page, then they already have it installed or know where to find that information :-)

File: pages/common/vue-init.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+> Official templates include: webpack, webpack-simple, browserify, browserify-simple, simple.
```
Comment (sbrl) [inline pages/common/vue-init.md]: Perhaps this will read better?
```suggestion
> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
```

File: pages/common/vue-init.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# vue-init
+
+> Legacy init command.
```
Comment (sbrl) [inline pages/common/vue-init.md]: Perhaps this will read better?
```suggestion
> Legacy init subcommand for the Vue.js framework.
```

File: pages/common/vue-prototyping.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# vue-prototyping
```
Comment (sbrl) [inline pages/common/vue-prototyping.md]: This page contains multiple subcommands. Maybe we could split `serve` and `build` into their own sub-pages?

File: pages/common/vue.md [hunk 6]
```diff
@@ -1,17 +1,12 @@
 # vue-cli
```
Comment (sbrl) [inline pages/common/vue.md]: Could we update this page title to match others more closely please?
```suggestion
# vue
```

File: pages/common/vue-prototyping.md (hunk 5 above)
Comment (KsRyY) [reply pages/common/vue-prototyping.md]: Thanks for your suggestion, applying changes.

File: pages/common/vue-init.md (hunk 2 above)
Comment (KsRyY) [reply pages/common/vue-init.md]: Applied changes to this.

File: pages/common/vue-prototyping.md (hunk 5 above)
Comment (KsRyY) [reply pages/common/vue-prototyping.md]: Changes commited to `patch-4`.

**Comment by KsRyY** [comment]
@sbrl applied your suggestions, thanks!

File: pages/common/vue-init.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# vue init
+
+> Legacy init subcommand for the Vue.js framework.
```
Comment (waldyrious) [inline pages/common/vue-init.md]: How about this?

```suggestion
> Legacy project initialization subcommand from the Vue.js framework.
```

(Or "of the Vue.js framework", if that reads better to you.)

File: pages/common/vue-init.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
+> More information: <https://cli.vuejs.org/guide/creating-a-project.html#pulling-2-x-templates-legacy>.
+
+- Create a new vue project:
```
Comment (waldyrious) [inline pages/common/vue-init.md]:
```suggestion
- Create a new Vue.js project:
```

File: pages/common/vue-init.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+`vue init {{template}} {{project_name}}`
+
+- Create a new project with a local template:
+
+`vue init {{path/to/template_directory}} {{project_name}}`
+
+- Create project using template on GitHub:
```
Comment (waldyrious) [inline pages/common/vue-init.md]:
```suggestion
- Create a project using a template on GitHub:
```

File: pages/common/vue-init.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+> Legacy init subcommand for the Vue.js framework.
+> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
+> More information: <https://cli.vuejs.org/guide/creating-a-project.html#pulling-2-x-templates-legacy>.
+
+- Create a new vue project:
+
+`vue init {{template}} {{project_name}}`
+
+- Create a new project with a local template:
```
Comment (waldyrious) [inline pages/common/vue-init.md]: Maybe reword to match the next example?

```suggestion
- Create a new project using a local template:
```

File: pages/common/vue-serve.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+# vue serve
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do painless prototyping.
```
Comment (waldyrious) [inline pages/common/vue-serve.md]: This description is exactly the same as the one for `vue build`. What's the difference between the two subcommands?

Comment (pxgamer) [reply pages/common/vue-serve.md]: Build will create a production release in the `dist` directory, and serve sets up a hot-reloading development server.

File: pages/common/vue-build.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
+# vue build
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do painless prototyping.
```
Comment (agnivade) [inline pages/common/vue-build.md]: Let us change "you to do painless" with "quick" to keep it formal and impartial.

File: pages/common/vue-init.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
+# vue init
+
+> Legacy init subcommand for the Vue.js framework.
+> Official templates include `webpack`, `webpack-simple`, `browserify`, `browserify-simple`, and `simple`.
```
Comment (agnivade) [inline pages/common/vue-init.md]: We can delete this line and include this information in the `vue init` command by writing it as - `{{webpack|webpack-simple..}}`

File: pages/common/vue-serve.md (hunk 11 above)
Comment (agnivade) [inline pages/common/vue-serve.md]: Same comment as vue build.

Comment (waldyrious) [reply pages/common/vue-serve.md]: Can we tweak the descriptions to make this distinction explicit, then?

File: pages/common/vue-build.md (hunk 12 above)
Comment (KsRyY) [reply pages/common/vue-build.md]: Thanks. Making the changes.

File: pages/common/vue-build.md [hunk 14]
```diff
@@ -0,0 +1,8 @@
+# vue build
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do quick prototyping.
```
Comment (agnivade) [inline pages/common/vue-build.md]: Please also remove "you to do" as I mentioned in my last comment. We prefer the tone in our pages to be in the 3rd person.

Comment (waldyrious) [reply pages/common/vue-build.md]: Agreed. In which case, "enables" could perhaps work better than "allows".

**Comment by sbrl** [comment]
Thanks for sticking with the review process on this one, @KsRyY - it usually doesn't take this long :P

File: pages/common/vue-init.md (hunk 13 above)
Comment (KsRyY) [reply pages/common/vue-init.md]: OK, thanks. Making the changes.

File: pages/common/vue-serve.md (hunk 11 above)
Comment (KsRyY) [reply pages/common/vue-serve.md]: Changes applied. Thanks for your suggestions!

File: pages/common/vue-init.md (hunk 7 above)
Comment (KsRyY) [reply pages/common/vue-init.md]: Changes applied. Thanks for your suggestions!

File: pages/common/vue-build.md (hunk 14 above)
Comment (KsRyY) [reply pages/common/vue-build.md]: Changes applied. Thanks for your suggestions!

**Comment by KsRyY** [comment]
@agnivade @waldyrious Applied all of your changes. Thanks.

File: pages/common/vue-init.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+# vue init
+
+> Legacy project initialization subcommand of the Vue.js framework.
+> More information: <https://cli.vuejs.org/guide/creating-a-project.html#pulling-2-x-templates-legacy>.
+
+- Create a new Vue.js project:
```
Comment (waldyrious) [inline pages/common/vue-init.md]: I'd mention the template parameter in the description, to make it explicit, and to match the structure of the following two examples. For example:

```suggestion
- Create a new Vue.js project using one of the default templates:
```

Comment (waldyrious) [reply pages/common/vue-init.md]: Also, the language should be similar in all three cases, so instead of:
>\- Create a new Vue.js project...
>\- Create a new project...
>\- Create a project...

all three examples should start the same way, e.g.
>\- Create a new project...

**Comment by waldyrious** [comment]
Hey @KsRyY, thanks for the update! I left a final set of suggestions, but if you prefer we can do them ourselves to spare you the work, since this PR has dragged a bit already. 😅

**Comment by mebeim** [comment]
(sorry for that force push I was just checking if maintainer edits were allowed)

@agnivade @waldyrious I would suggest adding @waldyrious changes ourselves and then merge. Should we squash this or force-push to have 4 different commits? Second option doesn't look that simple.

**Comment by agnivade** [comment]
Squash it. Go ahead if you want to do the changes yourself.

**Comment by mebeim** [comment]
Done. Do you guys know if there's a less intrusive way to check if contributor edits are allowed? I mean, making a test commit on the user's repo seems a bit silly.

**Comment by agnivade** [comment]
The "edit file" option will be enabled if maintainer edits are allowed.

**Comment by mebeim** [comment]
@agnivade oh, damn that's sneaky. Thank you :slightly_smiling_face:
